src/component/ItemBox/index.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import styled from "styled-components";
+import Item from "../Item";
+
+const ItemSection = styled.div`
+  height: 1230px;
+  background-color: #f8f9fa;
+  padding-top: 48px;
+  padding-bottom: 48px;
+`;
+
+const Container = styled.div`
+  width: 1140px;
+  height: 1230px;
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const ItemBox = (props) => {
+  return (
+    <ItemSection>
+      <Container>
+        <Item
+          url="jeju.jpg"
+          content="This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer."
+          min="9 mins"
+        ></Item>
```

Comment:
```suggestion
      <Container>
        <Item
          url="jeju.jpg"
          content="This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer."
          min="9 mins"
        />
```

여기도 self-closing tag!